settings.gradle.kts

```
@@ -192,6 +192,7 @@ include(":instrumentation:apache-httpclient:apache-httpclient-4.3:testing")
 include(":instrumentation:apache-httpclient:apache-httpclient-5.0:javaagent")
 include(":instrumentation:apache-httpclient:apache-httpclient-5.2:library")
 include(":instrumentation:apache-shenyu-2.4:javaagent")
+include(":instrumentation:apolloconfig-apolloclient:apolloconfig-apolloclient-1.1:javaagent")
```

```suggestion
include(":instrumentation:apolloconfig-apolloclient-1.0:javaagent")
```
we usually don't add a parent directory when not needed.

---

In the latest main branch of apollo, I saw the changes of advise point, but I didn't know their release plan, so I split the subdirectory in advance and used the 2.3.0] closed interval

---

> In the latest main branch of apollo, I saw the changes of advise point, but I didn't know their release plan, so I split the subdirectory in advance and used the 2.3.0] closed interval

For this problem, I wouldn't add unnecessary directory in this PR. It can be done when necessary.